modules/UI/videolayout/RemoteVideo.js

```diff
@@ -230,6 +230,7 @@ RemoteVideo.prototype.remove = function () {
     if (this.container.parentNode) {
         this.container.parentNode.removeChild(this.container);
     }
+    UIUtils.validateTooltips();
```

Comment:
Why do you call `UIUtils.validateTooltips` here? Why do you call it here only? I believe that's something that deserves a source code comment.